### PR TITLE
Email user settings

### PR DESCRIPTION
**Pull request by daniel-samson**
## Description

This change moves the email settings to the user's profile. I have changed the settings so that it edits the focus (Users Module) instead of the current_user global. This means that system administrators can edit other users email settings. 

Also in this pull request I have remove the <slot> elements and replaced them with <span> elements. As I was experiencing an issue with them. I was unable to see the inputs inside them. This is related to issue #3232.

This change includes the code from gyula's pull request #1675 and #2469. I have fixed some of the issues like removing the done buttons. 

Also please review and merge pr #3307 as this pr contains the same content.

## How To Test This

* Rebuild Sass
* Repair and Rebuild 
* Rebuild JavaScript
* Test the email settings tab in the user's profile / user management edit view

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [x] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: include/javascript/jquery/jquery.sugarMenu.js [hunk 1]
```diff
@@ -212,11 +212,11 @@
               jNode.height(0).show().animate(_animation, slideDownSpeed, function () {
                 $(this).css('height', '');
                 setTimeout(function () {
-                  $('.subnav.ddopen').each(function (i, e) {
-                    if (!$(e).hasClass('upper')) {
-                      $(e).css('top', parseInt($(e).css('top')) + 10 + 'px');
-                    }
-                  })
+                  // $('.subnav.ddopen').each(function (i, e) {
```
Comment (samus-aran) [inline include/javascript/jquery/jquery.sugarMenu.js]: Remove don't comment out

File: include/phpmailer/class.phpmailer.php [hunk 2]
```diff
@@ -678,6 +678,7 @@ public function __destruct()
      * @param string $to To
      * @param string $subject Subject
      * @param string $body Message Body
+     * @param string $body Message Body
```
Comment (samus-aran) [inline include/phpmailer/class.phpmailer.php]: duplicate string

File: jssource/src_files/include/javascript/message-box.js [hunk 3]
```diff
@@ -0,0 +1,272 @@
+/**
+ * @package MessageBox
+ * @author Daniel Samson
```
Comment (samus-aran) [inline jssource/src_files/include/javascript/message-box.js]: Probably from previous PR - but again update to be more SA

File: modules/Emails/include/ComposeView/ComposeView.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+ * of this program must display Appropriate Legal Notices, as required under
+ * Section 5 of the GNU Affero General Public License version 3.
+ *
+ * In accordance with Section 7(b) of the GNU Affero General Public License version 3,
+ * these Appropriate Legal Notices must retain the display of the "Powered by
+ * SugarCRM" logo and "Supercharged by SuiteCRM" logo. If the display of the logos is not
+ * reasonably feasible for  technical reasons, the Appropriate Legal Notices must
+ * display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
+ */
+
```
Comment (samus-aran) [inline modules/Emails/include/ComposeView/ComposeView.php]: Need entry point validation?

File: themes/SuiteR/css/style.css [hunk 5]
```diff
@@ -5136,11 +5136,11 @@ td.inlineEditActive{
 /* Fix to address bug with popups overflowing in the emails module*/
 
 #editAccountDialogue_c{
-    height:90% !important;
+    /*height:90% !important;*/
```
Comment (samus-aran) [inline themes/SuiteR/css/style.css]: Remove dont't comment out.

**Comment by daniel-samson** [comment]
Updated PR to meet requested changes

**Pull request closed without merging**